hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/heatMap/heatmap.tsx

```diff
@@ -104,22 +150,22 @@ export class Heatmap extends React.Component<Record<string, object>, ITreeState>
     const regex = /^[a-zA-Z0-9_/]*$/;
     if (regex.test(value)) {
       validExpression = value;
-     }
+    }
     else {
       alert("Please Enter Valid Input Path.");
```

Comment:
@devabhishekpal  Thanks for working on this patch, as part of improvement, a minor suggestion , can we just display the text beside the textbox in red and disable any actionable UI components instead of showing annoying alert box.

Reply:
Of course